cibuildwheel/__main__.py

```diff
@@ -191,8 +191,11 @@ def main() -> None:
         archs_config_str = get_option_from_environment('CIBW_ARCHS', platform=platform, default='auto')
     archs = Architecture.parse_config(archs_config_str, platform=platform)
 
+    identifiers = get_build_identifiers(platform, build_selector, archs)
+
     if args.print_build_identifiers:
-        print_build_identifiers(platform, build_selector, archs)
+        for identifier in identifiers:
+            print(identifier)
         sys.exit(0)
```

Comment:
I chose to always return 0 if you print build identifiers. Technically, you could return 3 here too and use it in an api to ask if something works - but I don't think anyone is asking for that, and normal usage for this is by hand AFAIK.

Reply:
Agreed. I think this wouldn't be an error; you're just asking for a list of what would be built, right?

Reply:
Yes `0` is right, because it successfully printed.